docs/blog/2020-06-18-Gatsby-cli-survey-2/index.md

```diff
@@ -71,6 +71,6 @@ Requests for incremental builds and complaints about slow builds have decreased
 
 Thank you again to all who have responded to the survey so far; your responses help guide the focus of the community to answer real-world needs. If you haven't filled out the survey yet, you can run `gatsby feedback` at any time after upgrading your version of `gatsby` in a project's `package.json` file, and then rerunning dependency and install commands. We would love to hear from you!
 
-Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most, and then report results. (If you'd like to opt out, visit [https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out](docs/telemetry/#how-to-opt-out/)).
+Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most, and then report results. (If you'd like to opt out, visit [https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out](/docs/telemetry/#how-to-opt-out/)).
```

Comment:
Getting 404 I think it would be better if

```md
[https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out](https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-ou/)
```

Reply:
I checked the other links to the documentation from the blog and they are all in the form `/docs` without hostname, so I think we should stick to that. Both the blog and the rest of the docs are hosted on `www.gatsbyjs.org` (the URL of this page is https://www.gatsbyjs.org/blog/2020-06-18-Gatsby-cli-survey-2/)

Reply:
Ahh got it!